tests/test_return_class.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from src.return_class import *
+
+
+class Test_ReturnClass(unittest.TestCase):
```

Comment:
This class name shoould be without `_` like
```suggestion
class TestReturnClass(unittest.TestCase):
```